### PR TITLE
Fetch by slug and radius

### LINKED ISSUE #16: Support fetching by slug and radius
User story: when you enter on the site by clicking on specific search URL, user wants to load a form with pre-filled locations and map with rendered data points as fast as possible. 

GraphQL should therefore support fetching all locations from slug and radius (which we get from this URL for example: https://www.kiwi.com/en/search/bratislava-slovakia-169km/56.25-19.42-250km) in order to support that. For now we filter them by cities only, it is possible that we would use stations/airports in the future as well for specific cases.

### PR DESCRIPTION
**Pull request by michalsanger**
close #16

### CODE REVIEW

**Comment by michalsanger** [comment]
May I ask for review plz? @jalbertsr

File: src/location/services/SlugRadiusParser.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  radius: number,
+|};
+type GpsRadius = {|
+  gps: { lat: number, lng: number },
+  radius: number,
+|};
+
+export default function parse(slugRadius: string): SlugRadius | GpsRadius {
+  const radius = getRadius(slugRadius);
+  let slug = stripRadius(slugRadius, radius);
```
Comment (jalbertsr) [inline src/location/services/SlugRadiusParser.js]: why not `const` ?

Comment (michalsanger) [reply src/location/services/SlugRadiusParser.js]: Good catch, fixed.

Comment (tbergq) [reply src/location/services/SlugRadiusParser.js]: What about adding this to eslint?-
`eslint prefer-const` -> `This rule is aimed at flagging variables that are declared using let keyword, but never reassigned after the initial assignment.`

Comment (michalsanger) [reply src/location/services/SlugRadiusParser.js]: @tbergq good idea, could you make PR with that addition?

Comment (tbergq) [reply src/location/services/SlugRadiusParser.js]: Sure

**Pull request merged**